lamindb_setup/core/_settings_instance.py

```diff
@@ -424,16 +431,16 @@ def is_on_hub(self) -> bool:
     def _get_settings_file(self) -> Path:
         return instance_settings_file(self.name, self.owner)
 
-    def _persist(self) -> None:
-        assert self.name is not None
-
-        filepath = self._get_settings_file()
-        # persist under filepath for later reference
-        save_instance_settings(self, filepath)
-        # persist under current file for auto load
-        shutil.copy2(filepath, current_instance_settings_file())
-        # persist under settings class for same session reference
-        # need to import here to avoid circular import
+    def _persist(self, write: bool = True) -> None:
```

Comment:
Can you add a docstring or make this more explicit by calling the arg `save_to_disk` or similar?

Reply:
done